docs/05-reference/api/update-asset.mdx

```diff
@@ -140,3 +140,29 @@ curl -X PATCH https://livepeer.studio/api/asset/{id} \
 
 Returns a `200 OK` response with an object containing the updated properties
 indicates that the `asset` was successfully updated.
+
+## Update Access Control Policy
+
+A `playbackPolicy` `type` can be either:
+- public - No access control is applied to the asset
+- jwt - A JWT is required to playback the asset
+
+If you set your stream or asset's playback policy to jwt, you will need to decode and verify
+your stream or asset for playback.
```

Comment:
```suggestion
If you set your stream or asset's playback policy to JWT, you will need to sign a token with a signing key
and put it in the playback URL to be able to playback your stream.
```

We should also add a link here to the documentation that explains how to create a signing key and sign a JWT with it.